support-frontend/assets/helpers/internationalisation/shouldCollectStateForContribs.ts

```diff
@@ -0,0 +1,33 @@
+import type { CountryGroup, CountryGroupId } from './countryGroup';
+import {
+	AUDCountries,
+	Canada,
+	countryGroups,
+	UnitedStates,
+} from './countryGroup';
+
+export function shouldCollectStateForContributions(
```

Comment:
This is the same logic that's inside the `ContributionsState` component to determine whether to display anything; we should be able to re-use it there in the future.